sigs.yaml

```diff
@@ -510,6 +510,15 @@ sigs:
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-secrets-store-csi-driver
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/OWNERS
+    meetings:
+    - description: Secrets Store CSI Meeting (biweekly)
+      day: Thursday
+      time: "8:00"
+      tz: PT (Pacific Time)
+      frequency: biweekly
+      url: https://zoom.us/j/91272289538
+      archive_url: https://docs.google.com/document/d/1q74nboAg0GSPcom3kLWCIoWg43Qg3mr306KNL58f2hg/edit#
+      recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP0PCFJrlpV6_nR_j_3RtnwI
```

Comment:
I think this would be better under the main sig-auth meeting.  Otherwise this information is too buried IMO.